### PR TITLE
Scaffold footer

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: css/responsive/desktop-mid.styl [hunk 1]
```diff
@@ -1 +1,4 @@
-//copy styles from site.styl
\ No newline at end of file
+//copy styles from site.styl
+
+.container
+  width: 993px
```
Comment (varffs) [inline css/responsive/desktop-mid.styl:4]: are you just doing this so you can build on your mac?

Comment (mikeyrayvon) [inline css/responsive/desktop-mid.styl:4]: no. but what is desktop mid anyways?  

lets just leave this as-is and attend to it properly later.  the responsive approach needs some revision i think, no?

File: footer.php [hunk 2]
```diff
@@ -1,4 +1,119 @@
...
+      $twitter = IGV_get_option('_igv_social_options', '_igv_socialmedia_twitter');
+      $instagram = IGV_get_option('_igv_social_options', '_igv_socialmedia_instagram');
+    ?>
+      <div class="container">
+
+        <div class="row">
+
+          <div class="col col-l col-l-1 align-center">
+        <?php if (!empty($logo_id)) { ?>
+            <a href="<?php echo home_url(); ?>"><?php echo wp_get_attachment_image($logo_id); ?></a>
```
Comment (varffs) [inline footer.php]: no specified size for image

File: footer.php [hunk 3]
```diff
@@ -1,4 +1,119 @@
...
+              <input type="submit" value="Subscribe" name="subscribe" class="button">
+            </form>
+          </div>
+
+        <?php } ?>
+
+        <?php if (!empty($facebook)) { ?>
+
+          <div class="col col-l col-l-1 align-center">
+            <?php echo '<a href="' . $facebook . '"><img src="' . get_stylesheet_directory_uri() . '/img/dist/fb-icon.png"></a>'; ?>
```
Comment (varffs) [inline footer.php]: the social media images dont exist, but should be svg tho

File: footer.php [hunk 4]
```diff
@@ -1,4 +1,119 @@
...
+          </div>
+
+        <?php } ?>
+
+        <?php if (!empty($twitter)) { ?>
+
+          <div class="col col-l col-l-1 align-center">
```
Comment (varffs) [inline footer.php]: layout for social media is different to the design `material-fair-com-frontpage-beforepub-sketch-2.pdf` which is the latest PDF I can find in the box

File: footer.php [hunk 5]
```diff
@@ -1,4 +1,119 @@
...
+        <?php } ?>
+          </div>
+
+        <?php if (!empty($address)) { ?>
+
+          <div class="col col-l col-l-2">
+          <?php
+            _e('<p>Material Art Fair [:en](Office)[:es](Oficina)</p>');
+
+            echo wpautop($address);
```
Comment (varffs) [inline footer.php:36]: according to cas we shouldnt use `wpautop` but call the filters for the_content on wysiwyg meta. I'm not sure how much it matters but you can find reference code in recent repos

File: footer.php (hunk 4 above)
Comment (mikeyrayvon) [inline footer.php]: there isnt a pdf that reflects this layout.  i had to rework it on the fly..

File: footer.php (hunk 5 above)
Comment (varffs) [inline footer.php:36]: well the point i guess is that calling the filters does everything, autop just adds paragraphs

File: header.php [hunk 6]
```diff
@@ -114,8 +114,8 @@
         </div>
         <div class="col col-l col-l-2 flex-col justify-center align-center"> 
           <?php 
-              if ($apply_url) { 
-                echo '<a href="' . $app_login_url . '" class="col flex-col justify-center">';
+              if (!empty($apply_url)) {
```
Comment (varffs) [inline header.php:117]: 👍

File: footer.php (hunk 5 above)
Comment (mikeyrayvon) [inline footer.php:36]: what would we need from the_content filters other than p tags?

File: header.php [hunk 7]
```diff
@@ -78,8 +78,8 @@
   <header id="header" class="section">
     <div class="container">
       <div class="row">
-        <div class="col col-l col-l-2 align-center">
-        <?php if ($logo_id) { ?>
+        <div class="col col-l col-l-2 flex-col justify-center align-center">
+        <?php if (!empty($logo_id)) { ?>
           <a href="<?php echo home_url(); ?>"><?php echo wp_get_attachment_image($logo_id); ?></a>
```
Comment (varffs) [inline header.php]: this still needs image size, i missed first time sorry

File: footer.php (hunk 5 above)
Comment (mikeyrayvon) [inline footer.php:36]: ah, but this isnt coming from wysiwyg. its text area.  so i just neede the line breaks

**Comment by varffs** [comment]
inline code comments + I assume that Mailchimp happens later as a feature?

File: footer.php (hunk 4 above)
Comment (mikeyrayvon) [inline footer.php]: i'll update the design to reflect this tho.  i realized while scaffolding that my layout was illogical.  didnt fit a grid

File: footer.php (hunk 3 above)
Comment (mikeyrayvon) [inline footer.php]: do we include? or img src=file.svg ?

File: css/responsive/desktop-mid.styl (hunk 1 above)
Comment (varffs) [inline css/responsive/desktop-mid.styl:4]: yes def. ok for now tho sure

File: footer.php (hunk 5 above)
Comment (varffs) [inline footer.php:36]: ok kl

File: css/responsive/desktop-mid.styl (hunk 1 above)
Comment (mikeyrayvon) [inline css/responsive/desktop-mid.styl:4]: actually i was copying the value from line 145 of site.styl

File: footer.php (hunk 3 above)
Comment (varffs) [inline footer.php]: we can do either. i like inlined svgs for styling but for stuff like this its fine i'm sure just to src the file

File: css/responsive/desktop-mid.styl (hunk 1 above)
Comment (varffs) [inline css/responsive/desktop-mid.styl:4]: oh right the breakpoint?

File: footer.php (hunk 2 above)
Comment (mikeyrayvon) [inline footer.php]: `wp_get_attachment_image` gets wp default size 'thumbnail' by default. that fits the container ok

**Comment by mikeyrayvon** [comment]
u want me to inline comment, you mean?

yea mailchimp ajax feature later.

**Comment by varffs** [comment]
i meant the image stuff in my code commnets

File: css/responsive/desktop-mid.styl (hunk 1 above)
Comment (mikeyrayvon) [inline css/responsive/desktop-mid.styl:4]: yea the breakpoint that _you_ made.  what is that breakpoint for neways?

**Pull request merged**